tests/virtualization/xen/xl_stop.pm

```diff
@@ -24,19 +24,20 @@ use testapi;
 use utils;
 
 sub run {
-    my ($self) = @_;
-    my $hypervisor = get_required_var('QAM_XEN_HYPERVISOR');
+    select_console 'root-console';
```

Comment:
not needed before `select_serial_terminal()`

Reply:
multiple occurrences above

Reply:
I kept this as it is (as we agreed) - otherwise it failing because of there's `desktop` but 'cleared-console' is missing.